File: Java/Questions/Record_Breaker.java
import java.util.Scanner;

public class Record_Breaker {
    public static void main(String [] args){
        Scanner kb = new Scanner(System.in);
        int n = kb.nextInt();
        int[] arr = new int[n+1];
        arr[n] = Integer.MIN_VALUE;

        for(int i = 0; i < n; i++){
            arr[i] = kb.nextInt();
        }
        int mx = Integer.MIN_VALUE;
        int ans = 0;

        if (n == 1){
            System.out.println("1");
            System.exit(0);
        }
        for(int i = 0; i < n+1; i++){
            if (arr[i] > mx && arr[i] > arr[i+1] ){
                ans++;
            }
            mx = Math.max(mx, arr[i]);
        }
        System.out.println("Record Breaking Days : "+ans);
    }
}

File: C++/Questions/record_breaker.cpp
#include<iostream>
#include<climits>
using namespace std;

int main(){
  

  int n;
  cin >> n;
  
  int arr[n+1];  
  arr[n] = -1;
  
  for (int i = 0; i < n; i++){
    cin >> arr[i];
  }

  int mx = INT_MIN;
  if ( n == 1){
    cout << "1" << endl;
    return 0;
  }
  int ans = 0;
  for (int i = 0; i < n; i++){
    if (arr[i] > mx && arr[i] > arr[i+1]){
      ans += 1;
    }
    mx = max(mx, arr[i]);
  }
  cout << ans << endl;
  return 0;

}
